src/main/java/com/zendesk/maxwell/schema/Table.java

```diff
@@ -144,6 +117,15 @@ private void renumberColumns() {
 			c.setPos(i++);
 		}
 	}
+
+	public void setDefaultColumnEncodings() {
```

Comment:
yeah, good call, I think `Table` should do the same thing that `ColumnDef` does, and only fallback to the database's default encoding on table creation.